pkg/phlare/phlare.go

```diff
@@ -523,6 +529,16 @@ func (f *Phlare) initAPI() (services.Service, error) {
 	return nil, nil
 }
 
+func (f *Phlare) initVersion() (services.Service, error) {
+	var err error
+	f.versions, err = apiversion.New(f.Cfg.Distributor.DistributorRing, f.logger, f.reg)
```

Comment:
not specifically happy that we use the distributor config but it's at least one guaranteed to exists and be valid.